types/node/ts4.8/http.d.ts

```diff
@@ -200,7 +200,7 @@ declare module "http" {
         auth?: string | null | undefined;
         // https://github.com/nodejs/node/blob/master/lib/_http_client.js#L278
```

Comment:
I would recommend removing this line if you're also going to remove it from the other one. The two dirs should be nearly identical (trying to fix that in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68356)